images/onboarding-bg.svg

```diff
@@ -0,0 +1,143 @@
+<svg width="1366" height="767" viewBox="0 0 1366 767" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Are these onboarding images intended to be part of this PR? Maybe a bad merge?

Reply:
Yeah looks like this overwrites the work we did to theme this.